modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm

```diff
@@ -22,7 +22,7 @@
 			user.allowed_turfs += list("pawprint", "hoofprint", "clawprint")
 
 		//species & taurs
-		if(islizard(user) || issynthliz(user) || HAS_TRAIT(user, TRAIT_ASH_ASPECT))
+		if(islizard(user) || HAS_TRAIT(user, TRAIT_ASH_ASPECT))
```

Comment:
So synthetics won't be allowed to use the smoke turf emote?

Reply:
There's a quirk for it.